pkg/logs/message/origin.go

```diff
@@ -49,11 +49,18 @@ func (o *Origin) TagsPayload() []byte {
 	if o.LogSource.Config.SourceCategory != "" {
 		tagsPayload = append(tagsPayload, []byte("[dd ddsourcecategory=\""+o.LogSource.Config.SourceCategory+"\"]")...)
 	}
+	var tags string
 	if o.LogSource.Config.Tags != "" {
-		tags := o.LogSource.Config.Tags
-		if len(o.tags) > 0 {
+		tags = o.LogSource.Config.Tags
+	}
+	if len(o.tags) > 0 {
+		if tags != "" {
 			tags = tags + "," + strings.Join(o.tags, ",")
+		} else {
+			tags = strings.Join(o.tags, ",")
 		}
```

Comment:
what about 
```
if tags != "" {
    tags = tags + ","
}
tags = tags + strings.Join(o.tags, ",")
```

Reply:
Alternative would be to define `all_tags` as `[o.LogSource.Config.Tags, strings.Join(o.tags, ",")]` and do a join there